src/components/Nav/Nav.module.css

```diff
@@ -0,0 +1,86 @@
+/* Copyright 2023 The Matrix.org Foundation C.I.C.
```

Comment:
Is it right that these copyright notices are for the foundation rather than New Vector Ltd?